examples/jsm/webxr/XRControllerModelLoader.js

```diff
@@ -0,0 +1,286 @@
+/**
+ * @author Nell Waliczek / https://github.com/NellWaliczek
+ * @author Brandon Jones / https://github.com/toji
+ */
+
+import {
+	Mesh,
+	MeshBasicMaterial,
+	Object3D,
+	Quaternion,
+	SphereGeometry,
+} from "../../../build/three.module.js";
+
+import {
+	Constants as MotionControllerConstants,
+	fetchProfile,
+	MotionController
+} from 'https://cdn.jsdelivr.net/npm/@webxr-input-profiles/motion-controllers@0.1.0/dist/motion-controllers.module.js'
```

Comment:
Are we planning on using this from within @webxr-input-profiles/viewer?  If so, we can't hardcode this path or it won't build against the @webxr-input-profiles/motion-controllers package in the same repo.